<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegiWindow</class>
 <widget class="QMainWindow" name="RegiWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>451</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>538</width>
    <height>451</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>538</width>
    <height>451</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>531</width>
      <height>458</height>
     </size>
    </property>
    <property name="text">
     <string>submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Your account name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Input your account password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Verify your password </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameText">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordText">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="verifyText">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="backToLoginWindowButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Back </string>
    </property>
   </widget>
   <widget class="QLabel" name="errorText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>538</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
